examples/README.md

```diff
@@ -28,8 +28,32 @@ Run the example:
 ```
 ./examples/agent/bin/run.sh
 ```
-  
-## FireLens on ECS EC2
+
+## Docker
+
+With Docker images, using the `awslogs` log driver will send your container logs to CloudWatch Logs. All you have to do is write to STDOUT and your EMF logs will be processed.
+
+[Official Docker documentation for `awslogs` driver](https://docs.docker.com/config/containers/logging/awslogs/)
+
+## ECS and Fargate
+
+With ECS and Fargate, you can use the `awslogs` log driver to have your logs sent to CloudWatch Logs on your behalf. After configuring your task to use the `awslogs` log driver, you may write your EMF logs to STDOUT and they will be processed.
+
+[ECS documentation on `awslogs` log driver](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/using_awslogs.html)
+
+## Fluent Bit and Fluentd
+
+Fluent Bit can be used to collect logs and push them to CloudWatch Logs. After configuring the Amazon CloudWatch Logs output plugin, you may write your EMF logs to STDOUT and they will be processed.
+
+[Getting Started with Fluent Bit](https://docs.fluentbit.io/manual/installation/getting-started-with-fluent-bit)
+
+[Amazon CloudWatch output plugin for Fluent Bit](https://docs.fluentbit.io/manual/pipeline/outputs/cloudwatch)
+
+## Previous Examples
+
+The following examples are still valid examples and can still be used, but they predate the release of the EMF auto-detection feature.
```

Comment:
Is there a reason why we want to mention 
> but they predate the release of the EMF auto-detection feature

If they are still valid then it be fine to just use them. Otherwise this might confuse customer as they might not know what auto-detection means.

Reply:
Thanks, good call on the “confusion” part. Pushed a commit to remove that text.